pages/artwork-info/[id].js

```diff
@@ -0,0 +1,21 @@
+import ArtworkInfo from "@/components/ArtworkInfo/index.js";
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+export default function ArtworkInfoPage({ artworks }) {
+  const router = useRouter();
+  const { id } = router.query;
+  const currentArtworkInfo = artworks.items.find((item) => item.id === id);
+
+  if (!currentArtworkInfo) {
+    return null;
+  }
+
+  return (
+    <>
+      <Link href={`/`}>Go back to Artworkslist</Link>
+      <h1>ArtworkInfoPage</h1>
+      <ArtworkInfo artworks={{ items: [currentArtworkInfo] }} />
```

Comment:
i think you are making it a bit more complicated than you have to :) 

this should work here and you can simplify the ArtworkInfo Component
```suggestion
      <ArtworkInfo artworks={currentArtworkInfo} />
```